summer_meal_sites_2020/summer_meal_sites_2020_scraper.py

```diff
@@ -0,0 +1,84 @@
+import os
+import sys
+from datetime import datetime
+import re
+import numpy as np
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+from pymongo import MongoClient, errors
+from tqdm import tqdm
+
+_i = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+if _i not in sys.path:
+    # add parent directory to sys.path so utils module is accessible
+    sys.path.insert(0, _i)
+del _i  # clean up global name space
+from shared_code.utils import (
+    check_similarity, locate_potential_duplicate,
+    insert_services, get_mongo_client
+)
+from shared_code.base_scraper import BaseScraper
+
+class Summer_Meal_Sites_2020_Scraper(BaseScraper):
+    def grab_data(self):
+        #Get the data
+        data = requests.get('https://opendata.arcgis.com/datasets/9efd2e8ba3104b88921b06fa3f70defb_0.geojson')
+        data = data.json()
+        data = data['features']
+        new_data = []
+        for item in data:
+            if item['properties']:
+                new_data.append(item['properties'])
+        df = pd.DataFrame(new_data)
+
+        #Extract useful cols
+        df = df.drop([ 'contactFirstName', 'contactLastName', 'contactPhone',
+       'sponsoringOrganization', 'startDate', 'endDate', 'daysofOperation',
+       'comments', 'breakfastTime', 'lunchTime', 'snackTimeAM', 'snackTimePM',
+       'dinnerSupperTime', 'mealTypesServed', 'cycleNumber', 'RecordStatus', 'FNSID', 'Created', 'Season', 'County', 'siteAddress2','ext'],
+        axis = 1)
+
+        #Removing schools, and non-homeless related resources
+        filter = df['siteName'].str.contains('(school|middle|elementary|high|academy|Academy|learn|Boy|Girl|Fire|Bus|College|Apartment|Route|Magnet)', flags = re.IGNORECASE)
+        df = df[~filter]
```

Comment:
better if we maintain a scraper constant list these keywords to look for in siteName and use it in filter as `"|".join(ignore_resource_with_keywords_list)`

We can then easily maintain such a list.